m/main_glider_data_processing_dt.m

```diff
@@ -233,6 +252,7 @@
   netcdf_l0_file = strfstruct(config.paths_local.netcdf_l0, deployment);
   netcdf_l1_file = strfstruct(config.paths_local.netcdf_l1, deployment);
   netcdf_l2_file = strfstruct(config.paths_local.netcdf_l2, deployment);
+  matlab_l2_file = strfstruct(config.paths_local.matlab_l2, deployment);
```

Comment:
No, I do everything by command line on my local machine and I push once in a while. But because I merged the branch with the master for the PR (the problem we discussed before), I am struggling with keeping my local version working (with my local paths) and updating the PR.

So everytime I must merge  my branch (but not all the functions) to the master to update the PR.